StepDefinitions/AddlangStepDefinitions.cs

```diff
@@ -0,0 +1,139 @@
+using OpenQA.Selenium;
+using System;
+using SpecProj2.Pages;
+using TechTalk.SpecFlow;
+using System.Collections.Generic;
+using TechTalk.SpecFlow;
+using OpenQA.Selenium.Chrome;
+using NUnit.Framework;
+using System.Reflection.Emit;
+using OpenQA.Selenium.Support.UI;
+using SpecProj2.Utilities;
+
+namespace SpecProj2.StepDefinitions
+{
+    [Binding]
+    public class AddlangStepDefinitions : CommonDriver
+    {
+        Login Loginobj = new Login();
+        Addlanguage Addlanguageobj = new Addlanguage();
+              
+
+        [Given(@"User logs into Mars Portal")]
+        public void GivenUserLogsIntoMarsPortal()
+        {
+            Loginobj.loginstep();
+        }
+        [When(@"User creates a new Language record '([^']*)' '([^']*)'")]
+        public void WhenUserCreatesANewLanguageRecord(string language, string level)
+        {
+            Addlanguageobj.cleardata();
```

Comment:
you should clear all data, only once  before any tests start, and then clean up test data only which the test generates